src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -637,3 +637,42 @@ def test_fw_codeblock():
         "    a=1\n"
         "PRINT *, 'I am a code block'\n"
         "    PRINT *, 'with more than one line'\n" in result)
+
+
+@pytest.mark.xfail(reason="issue #430 : module name should be specified")
+def test_module_name(monkeypatch):
```

Comment:
`monkeypatch` is not used and pylint complains about a missing import.

Reply:
Well observed.